dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
Hhmm may be I am missing something here but I am not seeing any property that is being used in the nodes and connections (in render method).

But the reason I am arguing is we are adding arbitrary properties to dag state which will be a problem when we start using types(flow) and define the exact structure of the state. If we need to add any property to the state it should be used in rendering and we need to be able to reason it from the store. Based on this I don't know what is being added to the state and what is being used to render them same. I'm sorry I am not comfortable with this change (and the change in the `addNode` function.)